### PR TITLE
Rework more controllers

### PR DESCRIPTION
**Pull request by GuilhemN**

### CODE REVIEW

File: service_container/autowiring.rst [hunk 1]
```diff
@@ -204,24 +205,25 @@ subsystem isn't able to find itself the interface implementation to register:
...
-        $container->register('rot13_transformer', 'Acme\Rot13Transformer');
-        $container->autowire('twitter_client', TwitterClient::class);
+        $container->register(Rot13Transformer::class);
+        $container->autowire(TwitterClient::class);
 
-The autowiring subsystem detects that the ``rot13_transformer`` service implements
-the ``TransformerInterface`` and injects it automatically. Even when using
-interfaces (and you should), building the service graph and refactoring the project
-is easier than with standard definitions.
+The autowiring subsystem detects that the ``Rot13Transformer::class`` service
```
Comment (xabbuh) [inline service_container/autowiring.rst]: Using the `class` constant here reads confusing. I would use the FQCN instead.

File: service_container/autowiring.rst [hunk 2]
```diff
@@ -204,24 +205,25 @@ subsystem isn't able to find itself the interface implementation to register:
...
+implements the ``TransformerInterface`` and injects it automatically. Even when
```
Comment (xabbuh) [inline service_container/autowiring.rst]: Don't we miss the alias for the `AppBundle\TransformerInterface`?

Comment (GuilhemN) [inline service_container/autowiring.rst]: Autowiring using types still works but is deprecated (`TransformerInterface` is detected on `Rot13Transformer`), do you mind if we reword this more deeply this later in another PR? Or maybe I can simply add a note that it is deprecated?

File: service_container/autowiring.rst (hunk 1 above)
Comment (GuilhemN) [inline service_container/autowiring.rst]: ok, updated :+1:

File: service_container/autowiring.rst (hunk 2 above)
Comment (xabbuh) [inline service_container/autowiring.rst]: We already have the alias a bit further down. Can we maybe just reorder the sections here?

**Comment by GuilhemN** [comment]
I reworked the autowiring article to focus on fqcn aliases.

**Comment by weaverryan** [comment]
Thanks @GuilhemN! I had not gotten to these articles yet, so it's perfect! I will review the autowire article a bit further... because it's really important... but  Idefinitely wanted to start by merging your changes :). Cheers!

File: service_container/autowiring.rst [hunk 3]
```diff
@@ -182,18 +199,22 @@ And update ``TwitterClient`` to depend of this new interface::
...
 
     .. code-block:: yaml
 
         services:
-            rot13_transformer:
-                class: Acme\Rot13Transformer
+            AppBundle\Rot13Transformer: ~
+
+            # the ``AppBundle\Rot13Transformer`` service will be injected when
+            # a ``AppBundle\TransformerInterface`` type-hint is detected
```
Comment (xabbuh) [inline service_container/autowiring.rst]: an

Comment (GuilhemN) [inline service_container/autowiring.rst]: see https://github.com/symfony/symfony-docs/pull/7914

**Pull request merged**